### DISCUSSION TITLE
bwipjs-api.metafloor.com server.js

### DISCUSSION THREAD

**Discussion by ct7abs** [metafloor/bwip-js#318] [Q&A]
Hi.
Is it possible to share the complete site (or the server.js)?
Ivé tryed my own node.js server, but it doens work properly with Crystal Reports, but this one, works great....
If possible ...
Tnx

**Comment by metafloor** [comment]
Can you explain what is not working for you? The `examples/server.js` is a straight forward example of how to use as a server.

**Reply by ct7abs** [reply]
Hi!
With your server.js (i assume that) it work perfectly in SAP and Crystal Reports. With the server.js example, it does not show the correct QR, or doesnt show anything at all, when working locally (and it stops a print service that we have)
Tryed everything (but, i'm no programmer....)

**Comment by metafloor** [comment]
Most likely you have a firewall permissions issue that is blocking the service.  Can you reach the service from a browser on the machine where CR is running?

**Reply by ct7abs** [reply]
Yes. it can be reached...

"http://bwipjs-api.metafloor.com/?bcid=gs1qrcode&text=(21)" + ToText({@as}) + "&scale=2&backgroundcolor=FFFFFF"
//"http://nodejs.xxxxxx.local/?bcid=gs1qrcode&text=(21)" + ToText({@as}) + "&scale=2&backgroundcolor=FFFFFF"

One works perfectly, the other, does not... And i've tryed with, and without firewall....
